docs/docs/features/fast-database-resets.md

```diff
@@ -9,7 +9,7 @@ To reset the database between each unit test, Joist generates a `flush_database`
 await knex.select(knex.raw("flush_database()"));
 ```
 
-This is generated at the end of the `joist-migation-utils` set only if `ADD_FLUSH_DATABASE` environment variable is set, i.e. this function should never exist in your production database. It is only for local testing.
+This is generated at the end of the `joist-codegen`, which should only be invoked against local development databases, i.e. this function should never exist in your production database. It is only for local testing.
```

Comment:
I moved `flush_database` to be created as the last step of codegen, instead of the last step of migrations.

The primary reason is that previously we'd copy/pasted the `isEntityTable` code between `joist-codegen` and `joist-migration-utils`, and having different definitions of `isEntityTable` for each of those is bad.

I could have moved `isEntityTable` into `joist=utils` or something, but dunno, actually kind of like running it as the last step of codegen, b/c it's already something that is "something you'd only run locally", and also it means we can easily re-use the pg-structure-driven metadata, instead of having to load it once for the migrations (that only used it for creating the flush function) and once for codegen (which of course just inherently needs it).